common/scala/src/main/scala/whisk/common/TransactionId.scala

```diff
@@ -215,6 +223,7 @@ object TransactionId {
   val loadbalancer = TransactionId(-120) // Loadbalancer thread
   val invokerHealth = TransactionId(-121) // Invoker supervision
   val controller = TransactionId(-130) // Controller startup
+  val longTransactionWarningTime = 60 // Threshold in seconds to log long-running transactions
```

Comment:
`val longTransactionWarningTime = 60.seconds.toMillis` and then you can remove the arbitrary `* 1000` above.